src/app/core/app.component.ts

```diff
@@ -26,6 +27,9 @@ export class AppComponent implements OnInit, OnDestroy {
     this.authService.failure$,
     this.sessionService.userProfileError$,
     this.localeService.currentLangError$,
+  ).pipe(
+    // eslint-disable-next-line no-console
+    tap(err => console.error(`fatal: ${err.message}`))
```

Comment:
Yes, this is console logging of the source of the error. 
Actually, without that, it was a bit "ridiculous": you had this "something went wrong" modal displayed but not a single hint of what happened. Instead of letting the service write to the console themselves, I let the component write the error itself.   Will be very useful if a customer reports us such a problem.

Reply:
If what we are looking for are hints, then including the stack trace would help us more: `console.error('fatal:', err)` is enough